app/assets/javascripts/oxalis/view/right-menu/tree_hierarchy_view.js

```diff
@@ -210,6 +262,25 @@ class TreeHierarchyView extends React.PureComponent<Props, State> {
     }
   };
 
+  getNodeStyleClassForBackground = (id, isGroup = false) => {
+    if (isGroup) {
+      if (this.props.selectedTreeGroups.includes(id) && this.props.activeGroupId === id) {
```

Comment:
Please write the results of the `includes` calls into local variables to avoid redundant computations.